pricing.js

```diff
@@ -2,35 +2,43 @@ module.exports.calculateProductPrice = function (product, employee, coverageLeve
   var price = 0
   var dollarsOff = 0
 
-  switch (product.type) {
-    case 'volLife':
-      for (var i = 0; i < coverageLevels.length; i++) {
-        var coverageAmount = coverageLevels[i].coverage
+  function volLifePrice() {
+    for (var i = 0; i < coverageLevels.length; i++) {
+      var coverageAmount = coverageLevels[i].coverage
 
-        price += (coverageAmount / product.cost.costDivisor) * product.cost.price
-      }
+      price += (coverageAmount / product.cost.costDivisor) * product.cost.price
+    }
 
-      if (product.employerContribution.mode === 'dollar') {
-        price = price - product.employerContribution.contribution
-      } else {
-        dollarsOff = price * (product.employerContribution.contribution / 100)
-        price = price - dollarsOff
-      }
+    if (product.employerContribution.mode === 'dollar') {
+      price = price - product.employerContribution.contribution
+    } else {
+      dollarsOff = price * (product.employerContribution.contribution / 100)
+      price = price - dollarsOff
+    }
+    return parseInt(price * 100) / 100
```

Comment:
Since line 18 and 32 is repeated, and technically is a separate concept, formatting the price, I would pull that out into a separate function as well. Seems small but remember we want our functions to do one thing. Calculating the price and formatting it are two different things.